lib/errors.js

```diff
@@ -32,10 +32,6 @@ module.exports = {
         message: 'Expected `options.app` to be a non-empty string.\n' +
             'Must specify an app for ringpop to work.\n'
     }),
-    DenyJoinError: TypedError({
```

Comment:
note: it might be useful to keep all errors in a file. This means that other libraries can `require('ringpop/lib/errors').DenyJoinError.type` or something.